packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -255,6 +254,18 @@ export class PKPBase<T = PKPBaseDefaultParams> {
     }
   }
 
+  public async authenticate(): Promise<void> {
```

Comment:
Would be good to have JSDoc on this if it's part of our public API, especially if we're going to leave the existing `controllerSessionSigs` intact if our call to `getSessionSigs()` returns a falsy value, explaining why that is

Reply:
added doc header